vrayGolaem/vraygolaem.cpp

```diff
@@ -147,6 +148,12 @@ INode *getNode(VRayGolaem *golaem) {
 	return node;
 }
 
+//------------------------------------------------------------
+bool fileExists(const CStr& pathname)
+{
+	return ( _access( pathname.ToCStr(), 0 ) == 0 ) ;
```

Comment:
This will probably not work with Unicode file names. Maybe using _waccess will be better.